azure-pipelines.yml

```diff
@@ -17,3 +18,59 @@ jobs:
       gradleOptions: '-Xmx3072m'
       publishJUnitResults: false
       tasks: 'assembleAndroidTest'
+
+  - bash: |
+      #!/usr/bin/env bash
+
+      uname -a
+      $ANDROID_HOME/tools/bin/sdkmanager --version
+
+      cd $ANDROID_HOME/tools
+
+      # Install all required sdk packages
+      echo "y" | bin/sdkmanager --update
```

Comment:
sdkmanager can take several argument to install, e.g:

--install "build-tools;$BUILD_TOOLS_VERSION" "platforms;android-19" "platform-tools"